feat(data_source): RedHat Security Database (#1949)
intel/cve-bin-tool: README.md

This data source consists of security advisories used by the GitLab dependency scanner. 
Amount of CVEs added from this data source is similar to OSV.

### [RedHat Advisory Database](https://access.redhat.com/security/data) (RSD)

> This data source contains CVEs pertaining to RedHat Products, a lot which are also included in NVD (duplicate CVEs are not reported multiple times).

Reviewer: ```suggestion
This data source contains CVEs pertaining to RedHat Products.  Many are also included in NVD (duplicate CVEs are not reported multiple times).
```

Reviewer: Minor grammatical fix.
